.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "config/peerpods/podvm/cloud-api-adaptor"]
+	path = config/peerpods/podvm/cloud-api-adaptor
+	url = https://github.com/openshift/cloud-api-adaptor/
+	branch = osc-release
```

Comment:
For reproducibility, shouldn't have an exact commit here? You mentioned that this causes conflicts during `go run`. I assume that would depend on the actual content, so maybe we want to make sure we know exactly which version we pick, not just a branch.

Reply:
I think we would have it whatever the commit ID.
The fix is in this PR, not on the CAA repo. It consists in modifying the Dockerfile for the operator's build to remove the podvm-payload sources (including that submodule folder). Without it, there is no risk of conflict, whatever the version we use from cloud-api-adaptor.
So no, we don't need to pin a specific version.